Spinner/Spinner.js

```diff
@@ -57,17 +63,29 @@ const SpinnerCore = kind({
 	},
 
 	computed: {
+		'aria-label': ({'aria-label': label, type}) => {
+			// TODO: These static values will need to be localized
+			return label || type === 'searching' ? 'Searching' : 'Loading';
+		},
 		className: ({styler, type}) => styler.append(type),
-		spinnerNodes: ({css, type}) => type === 'searching' ? createSpinnerNodes(12, css) : createSpinnerNodes(4, css)
+		spinnerNodes: ({styler, type}) => {
+			return Array.from({length: type === 'searching' ? 12 : 4}, (_, index) => (
```

Comment:
Couple changes here:

* Since the spinner nodes are only needed for this computed, including the code inline here (vs refactored into a separate method) improves the readability of the component
* Using `Array.from()` can simplify creating an array of components since it accepts a map fn as its second arg